### ISSUE TITLE
Recipe Book category editor not working

### ISSUE THREAD

**Issue by AlexanderMoebius** [WolfyScript/CustomCrafting#176]
### What happened?

The overlay does not show information about the category.
Even more /recipes is empty also it should not be.

Seems to be related to the thread "Recipe Book Editor Issue " (1030448536363487242) in Discord

### How can the bug be reproduced?

Use the recipe book.
Furthermore I used mysql for sync. Maybe this also be relevant

### Expected Behaviour?

Recipes should be shown in /recipes

### Plugin version

CustomCrafting 3.16.8.1 and WolfyUtils 4.16.7

### Server software

Paper 1.19.2 v215

### Submit issue

- [X] read the Wiki and especially the FaQ, and made sure there is no fix to your issue,
- [X] searched for and ensured there isn't already an open issue regarding this,
- [X] are running the latest version of your server software (Spigot, Paper, etc.), and plugins (WolfyUtilities, CustomCrafting).

**Comment by WolfyScript** [comment]
Yes, the recipe book config and recipe book editor got a complete overhaul (multiple, in fact, since 3.x).
To fix any issues with the recipe book and use the new config, you should delete the `recipe_book.json` (and `recipe_book.conf` if that already exists).
Then you'll have a fresh `recipe_book.conf`.

Regarding the categories. Not sure when I added it... if you just have a single category, you won't see the main menu, and it'll open the category automatically.
